IPs/RSKIPxxx.md

```diff
@@ -0,0 +1,262 @@
+---
+rskip: xxx
+title: Support for Bridging Ordinals
+description: 
+status: Draft
+purpose: Usa
+author: SDL (@sergiodemianlerner)
+layer: Core
+complexity: 2
+created: 2023-06-19
+---
+# Support for Bridging Ordinals
+
+|RSKIP          |xxx           |
+| :------------ |:-------------|
+|**Title**      |Support for Bridging Ordinals|
+|**Created**    |19-MAY-2023 |
+|**Author**     |SDL |
+|**Purpose**    |Usa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+|**discussions-to**     ||
+
+# **Abstract**
+
+This is a proposal for making Bitcoin ordinals and inscriptions available in Rootstock using the current Powpeg bridge. Once ordinals or inscriptions have representations in Rootstock,  they can be traded. Later they can be transfered back to any Bitcoin address.
+
+
+# **Motivation**
+
+Inscriptions are images, videos, recordings or text that is stored in the Bitcoin blockchain, can be owned by a user and the ownership can be transferred to another user using standard Bitcoin transactions. We can think that every satoshi ever minted in Bitcoin is a separate “virtual cheque” and has an unique serial number. Then those serial numbers are called Ordinals. Inscriptions are linked to specific ordinals, and move with those ordinals with every transaction that involves those virtual cheques. There has been a rising interest on inscriptions since their introduction.
```

Comment:
```suggestion
Inscriptions are images, videos, recordings, or text that are stored in the Bitcoin blockchain, can be owned by a user and the ownership can be transferred to another user using standard Bitcoin transactions. We can think that every satoshi ever minted in Bitcoin is a separate “virtual cheque” and has a unique serial number. Then those serial numbers are called Ordinals. Inscriptions are linked to specific ordinals, and move with those ordinals with every transaction that involves those virtual cheques. There has been a rising interest on inscriptions since their introduction.
```